connector/src/main/java/org/jboss/as/connector/subsystems/datasources/Constants.java

```diff
@@ -333,6 +337,20 @@ public void validateResolvedParameter(String parameterName, ModelNode value) thr
             .setAllowNull(true)
             .build();
 
+    static SimpleAttributeDefinition ENLISTMENT_TRACE = new SimpleAttributeDefinitionBuilder(ENLISTMENT_TRACE_NAME, ModelType.BOOLEAN)
+            .setXmlName(DataSource.Attribute.ENLISTMENT_TRACE.getLocalName())
+            .setAllowExpression(true)
+            .setDefaultValue(new ModelNode(true))
+            .setAllowNull(true)
+            .build();
+
+    static SimpleAttributeDefinition MCP = new SimpleAttributeDefinitionBuilder(MCP_NAME, ModelType.STRING)
+            .setAllowExpression(true)
+            .setAllowNull(true)
+            .setDefaultValue(new ModelNode("org.jboss.jca.core.connectionmanager.pool.mcp.SemaphoreArrayListManagedConnectionPool"))
```

Comment:
AIUI the default is supposed to be org.jboss.jca.core.connectionmanager.pool.mcp.SemaphoreConcurrentLinkedQueueManagedConnectionPool